src/components/SchemaViewer.svelte

```diff
@@ -1,11 +1,20 @@
 <script>
+  import page from "page";
   import SchemaNode from "./SchemaNode.svelte";
   import FilterInput from "./FilterInput.svelte";
 
   export let app;
   export let nodes = [];
+  export let queryString;
```

Comment:
If you implement my suggestion below, I think we can remove any concept of "query strings" from this component, and just focus on an interface to setting/handling changes to a searchText parameter.

```suggestion
  export let searchText;
```